src/program.js

```diff
@@ -167,6 +167,11 @@ Example: $0 --help run.
         demand: true,
         type: 'string',
       },
+      'timeout' : {
+        describe: 'A length in millesconds to give up after if the add-on ' +
+                  'hasn\'t been signed',
```

Comment:
> Number of milleseconds to wait before giving up. Default: 2 minutes.

I think setting the default here increases coupling with the https://github.com/mozilla/sign-addon library. Assuming that changing the timeout there would need a change here as well to maintain consistency/uniformity.

Let me know what you think @kumar303